console/console-init/ui/mock-console-server/mock-console-server.js

```diff
@@ -2155,7 +2155,7 @@ createIotDevice("iotProjectFrance", {
     },
     status: getIotDeviceStatusSection()
   }),
-  credentials: []
+  credentials: "[]"
```

Comment:
You should agree with Ajay on which one is correct, as he changed that a couple of days ago :P